From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Tom Levy <tomlevy93@gmail.com>
Date: Fri, 24 Jul 2020 09:33:23 +1200
Subject: Revert "Modify domserver Dockerfile to build from source instead of
 release tarball"

The judgehost Dockerfiles will also be added soon. Changing them to
build from source instead of release tarball as was done with
domserver is too much work, so that strategy will be dropped.

Instead, a separate Dockerfile will be added to build a release
tarball from source.

This reverts commit d5a2ec8aee62b84cdffc07872680f64efa29d008.
---
 docker/domserver/Dockerfile | 11 ++++-------
 1 file changed, 4 insertions(+), 7 deletions(-)

diff --git a/docker/domserver/Dockerfile b/docker/domserver/Dockerfile
index b218ece97..6a54c5480 100644
--- a/docker/domserver/Dockerfile
+++ b/docker/domserver/Dockerfile
@@ -32,11 +32,8 @@ RUN php composer-setup.php \
     && mv /composer.phar /usr/local/bin/composer
 
 # Add DOMjudge source code and build script
-COPY . /domjudge-src/domjudge
-ADD docker/domserver/build.sh /domjudge-src
-
-# Bootstrap the configure/build system
-RUN make -C /domjudge-src/domjudge dist
+ADD domjudge.tar.gz /domjudge-src
+ADD domserver/build.sh /domjudge-src
 
 # Build and install domserver
 RUN /domjudge-src/build.sh
@@ -73,7 +70,7 @@ RUN php composer-setup.php \
 # Copy domserver directory and add script files
 COPY --from=domserver-build /opt/domjudge/domserver /opt/domjudge/domserver
 COPY --from=domserver-build /opt/domjudge/doc /opt/domjudge/doc
-COPY ["docker/domserver/scripts", "/scripts/"]
+COPY ["domserver/scripts", "/scripts/"]
 
 # Add user, create PHP FPM socket dir, change permissions for domjudge directory and fix scripts
 RUN useradd -m domjudge \
@@ -88,7 +85,7 @@ RUN useradd -m domjudge \
 CMD ["/scripts/start.sh"]
 
 # Copy supervisor files
-COPY ["docker/domserver/supervisor", "/etc/supervisor/conf.d/"]
+COPY ["domserver/supervisor", "/etc/supervisor/conf.d/"]
 
 # Expose HTTP port
 EXPOSE 80
-- 
2.20.1

